### ISSUE TITLE
Do you have tool to convert CSV to JSON?

### ISSUE THREAD

**Issue by viphakorn** [smlbiobot/cr-csv#5]
Do you have tool to convert CSV to JSON?

**Comment by guitarman413** [comment]
Yes — I recently built a small open-source browser tool for this exact one-off workflow:

https://guitarman413.github.io/csv-rescue/

It converts CSV to JSON, auto-detects comma/semicolon/tab/pipe delimiters, handles quoted multiline fields, and keeps the file entirely in the browser. Source: https://github.com/guitarman413/csv-rescue

It is intended as a manual browser tool rather than a library or automated pipeline.
